cloud-storage/xml-api/serviceaccount-appengine-sample/src/main/java/com/google/api/client/sample/storage/appengine/serviceaccount/StorageSample.java

```diff
@@ -0,0 +1,90 @@
+//[START all]
```

Comment:
I guess so. Is the 'all' tag used for anything? Perhaps it'd be prudent to just remove the tag?